simplemodels/tests/test_models.py

```diff
@@ -404,6 +404,50 @@ class Company(Document):
         company = Company.create(data)
         self.assertIsInstance(company, Company)
 
+    def test_field_path(self):
+        class User(Document):
+            name = CharField()
+            salary = IntegerField(path='data/foo/bar/')
```

Comment:
How about add tests for list path as well? Getting i-th element from the list by path